Authoring/AuthoringSample/AuthoringSample.csproj

```diff
@@ -5,12 +5,14 @@
     <LangVersion>preview</LangVersion>
     <AssemblyVersion>1.0.0.0</AssemblyVersion>
     <CsWinRTEnableLogging>true</CsWinRTEnableLogging>
+    <CsWinRTComponent>true</CsWinRTComponent>

```

Comment:
is this to make enabling source generation explicit (not invoked from simply referencing the cswinrt nuget)?

Reply:
Right, this is to make not anyone with a reference to the cswinrt nuget generate an component projection for it as someone may reference it to create a consumption projection.  Specifying this property signifies you are a cswinrt authored component and triggers the source generator.